docs/source/model_design/risk_factors.rst

```diff
@@ -124,7 +124,25 @@ An term that is often used in causal inference is the **counterfactual**. The
 counterfactual refers to an alternate reality in which only a single variable 
 has changed and all else has remained exactly the same. 
 
-  For instance, say that we wanted to evaluate the causal realtionship between 
+  For instance, suppose you've finished work and you're trying to determine 
+  how to get down to the international district for dinner. You go to your 
+  maps application and ask **What is the quickest route to my destination?** 
+  It tells you that based on the current time, weather, day of the week, road 
+  work conditions, etc. that you should take the light rail.
+
+  So you leave the office, walk a few blocks to the westlake station, and 
+  catch the train. Two stops in, your train gets stopped behind a broken down 
+  bus. You decide to get off the train and walk the remaining distance. 
+  Arriving ten minutes late to dinner, you think to yourself **Would I have 
+  arrived on time if I took the bus instead?**
+
+As illustrated here, by definition the counterfactual question is impossible 
+to *directly* answer because it is a purely hypothetical question without 
+direct supporting evidence; we can only use what we know to make a *guess* at 
+what might have happened in the counterfactual scenario. Consider the 
```

Comment:
pompous literary quote suggestion: 

“We can never know what we want, because, living only one life, we can neither compare it with our previous lives nor perfect it in our lives to come." -Milan Kundera The Unbearable Lightness of Being